pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -335,6 +336,43 @@ func HasExpectedDNSNames(secretName, secretNamespace string, cert *x509.Certific
 }
 
 // CreateTrustedBundle creates a TrustedBundle, which provides standardized methods for mounting a bundle of certificates to trust.
-func (cm *certificateManager) CreateTrustedBundle(certificates ...certificatemanagement.CertificateInterface) certificatemanagement.TrustedBundle {
-	return certificatemanagement.CreateTrustedBundle(append([]certificatemanagement.CertificateInterface{cm.keyPair}, certificates...)...)
+func (cm *certificateManager) CreateTrustedBundle(includeSystemBundle bool, certificates ...certificatemanagement.CertificateInterface) (certificatemanagement.TrustedBundle, error) {
+	if includeSystemBundle {
+		systemCertificates, err := getSystemCertificates()
+		if err != nil {
+			return nil, err
+		}
+		certificates = append(certificates, systemCertificates...)
+	}
+	return certificatemanagement.CreateTrustedBundle(append([]certificatemanagement.CertificateInterface{cm.keyPair}, certificates...)...), nil
+}
+
+// certFiles is copied from the x509 package.
+// Possible certificate files; stop after finding one.
```

Comment:
Maybe include a link to the golang source here in case we need to update this list later?